src/main/java/net/imglib2/algorithm/convolution/Convolution.java

```diff
@@ -34,6 +34,7 @@
 	/**
 	 * Set the {@link ExecutorService} to be used for convolution.
 	 */
+	@Deprecated
```

Comment:
Without having studied the code too closely: does this work? I.e., if some existing downstream code is using `setExecutor`, won't that executor now be ignored, instead of used when executing? Or did I miss where this is handled in some descendant class?

Reply:
This method now works again.